apps/web-giddh/src/app/settings/integration/beneficiary/beneficiary.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="aside-pane">
+    <div class="form-header">
+        <div>
+            <h3><span>Beneficiary List</span></h3>
+        </div>
+    </div>
+    <button id="close" (click)="closeAsidePane()"><i class="icon-cross"></i></button>
+    <div class="aside-body">
+        <div class="aside-pane-form">
+            <form [formGroup]="addBeneficiaryForm" (ngSubmit)="saveBeneficiary()" *ngIf="isBeneficiaryFormVisible">
+                <div class="row">
+                    <div class="form-group  col-sm-6">
+                        <label>Beneficiary Name <sup class="text-danger">*</sup></label>
+                        <input type="text" class="form-control" [placeholder]="'Beneficiary'" name="bnfName" formControlName="bnfName" />
```

Comment:
Do use normal placeholder attribute instead of placeholder property, please change for all such places
```suggestion
                        <input type="text" class="form-control" placeholder="'Beneficiary'" name="bnfName" formControlName="bnfName" />
```